guides/process/mentoring/supporting_and_mentoring_other_engineers.md

```diff
@@ -0,0 +1,63 @@
+# Supporting and Mentoring other Engineers
+
+## Constructivist Learning Theory
+
+> Constructivist teaching is based on the belief that learning occurs as learners are actively involved in a process of meaning and knowledge construction as opposed to passively receiving information. Learners are the makers of meaning and knowledge.
+
+Our experience in mentoring and coaching has shown us we gain the most knowledge and understanding not through simply providing answers and solutions to questions or problems but instead by helping provide tools and information through which others can shape their own answers. We can often help another engineer when they ask a question by asking further questions to help frame thinking or providing potential sources of information that will provide further guidance to them.
+
+## Responding to questions
+
+When a colleague asks a question it's important that we try our best to make them feel listened to and understood, if they are new to this process it's probably worth explaining why we aren't directly answering their request. Explain the value we place on helping others to find the tools to solve their own problems rather than solving/answering directly for them
+
+### Clarifying the question
+
+Does the person asking the question have all the tools to adequately clarify the problem they are facing? If they can't it's probably best trying to guide them towards being able to better frame the problem they're facing and developing language around it.
+
+If they have the language but are struggling to explain the root cause you can try applying [5 whys](https://en.wikipedia.org/wiki/5_Whys) as a method of helping them focus in on it.
+
+### Guidance and leading over Answers
+
+Where possible when asked a question the best possible response is either a prompt or question that will help the learner explore their situation deeper without feeling they've been spoon fed or simply given a solution.
+
+We value providing others the tools to assemble their own learnings, to help with this we should phrase questions in a way that allows for open discussion and prompts consideration on the topic we're dealing with. Avoiding providing solutions or responding with closed Answers/Questions can shut down the ability for the asker to form their own understandings and views on the problem they are facing.
+
+### Sources of information
+
+Does the learner know the best place to look for information that could help answer their question? Do they know the correct terms to find this if they were to [Google it](#google-it)? If they don't guide them towards some good resources you know or if you're both struggling to find something that helps provide further understanding throw it out the team via the #slack-overflow channel in our slack.
```

Comment:
should there be a comma after "If they don't"? Also "throw it out _to_ the team"?